plugins/inputs/http_response/README.md

```diff
@@ -0,0 +1,45 @@
+# Example Input Plugin
+
+This input plugin will test HTTP/HTTPS connections.
+
+### Configuration:
+
+```
+# List of UDP/TCP connections you want to check
+[[inputs.http_response]]
+  ## Server address (default http://localhost)
+  address = "http://github.com"
+  ## Set response_timeout (default 10 seconds)
+  response_timeout = 10
+  ## HTTP Request Method
+  method = "GET"
+  ## HTTP Request Headers
```

Comment:
these headers should be specified as a map (see https://github.com/influxdata/telegraf/blob/master/plugins/inputs/httpjson/httpjson.go)